tests/tests_pytorch/test_cli.py

```diff
@@ -67,42 +63,6 @@ def mock_subclasses(baseclass, *subclasses):
         yield None
 
 
-@mock.patch("argparse.ArgumentParser.parse_args")
-def test_default_args(mock_argparse):
```

Comment:
Why are these CLI tests removed? Bad merge?

Reply:
@carmocca - Thanks for cross-checking, this comes from: https://github.com/Lightning-AI/lightning/pull/14597/. So probably not a bad merge.

Reply:
My bad. I missed that PR